admin/app/main/views/rotate_password.py

```diff
@@ -0,0 +1,31 @@
+import sys
+from flask import render_template, session, url_for, redirect
+from flask_login import current_user
+
+from app import user_api_client
+from app.main import main
+from app.main.forms import RotatePasswordForm
+from app.utils import redirect_to_sign_in
+
+
+@main.route('/change-password', methods=['GET', 'POST'])
```

Comment:
I'm now realising that the design doesn't call for it; but maybe we should prompt this through an email. see ticket for details so team can discuss. `NOT-143`

Reply:
Thanks @dagoof - have commented in ticket.